### PR TITLE
Do not provide onChange prop by default

### LINKED ISSUE #55: LinkState and valueLink bug
When I use the linkState mixin:

Uncaught Error: Invariant Violation: Cannot provide a valueLink and a value or onChange event. If you want to use value or onChange, you probably don't want to use valueLink.

Any idea?

### PR DESCRIPTION
**Pull request by faergeek**
This PR fixes #55

### CODE REVIEW

File: src/Textfield.js [hunk 1]
```diff
@@ -60,15 +52,19 @@ class Textfield extends React.Component {
...
             key: inputId,
             value: value,
-            onChange: this._handleChange,
             rows: rows,
             ...otherProps
-        });
+        };
+
+        if (onChange) inputProps.onChange =
+            event => this.props.onChange(event.target.value);
```
Comment (tleunen) [inline src/Textfield.js]: Creating an anomymous function every render is not optimal. It's way better to keep the `_handleChange` ;)

Comment (faergeek) [inline src/Textfield.js]: Sorry, I just got used to use `react-hot-loader` and now `react-transform-hmr`. So I avoid assigning methods on each instance.

Comment (tleunen) [inline src/Textfield.js]: Let me know when it's fixed and I'll merge

**Comment by faergeek** [comment]
Fixed

**Pull request merged**